GVRf/Framework/jni/shaders/material/unlit_shader.cpp

```diff
@@ -85,6 +85,11 @@ void UnlitShader::render(const glm::mat4& mvp_matrix,
         throw error;
     }
 
+    if (texture->getId() == 0) {
+        std::string error = "UnlitShader::render : texture with invalid Id";
```

Comment:
> Should this "if getId() == 0" check be applied to other renderers?

I think we'd be better off looking into ways to keep early-recycloing from happening at all.